xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -576,6 +576,7 @@ void CVideoPlayerAudio::Process()
         msg.player = VideoPlayer_AUDIO;
         msg.cachetotal = cachetotal;
         msg.cachetime = cachetime;
+        msg.next_pts = m_messageQueue.GetTimeBack();
         msg.timestamp = m_audioClock;
```

Comment:
allright - I have now disabled the sync for streams wich came form a change. Looks already smoother, but there is now a short black screen between the videos at second 20 - I'll investigate further - but the direction is right.